### PR TITLE
Query the size of the hw.intrnames mib instead of using of a fixed va…

### PR DESCRIPTION
**Pull request by MikaelUrankar**
It fixes the following error on FreeBSD aarch64:
```
getsysctl_simple; FREEBSD: sysctl(hw.intrnames...) failed: Cannot allocate memory
```

Relevant kernel code: https://github.com/freebsd/freebsd-src/blob/main/sys/kern/kern_intr.c#L1599-L1606

### CODE REVIEW

**Comment by MikaelUrankar** [comment]
> [CLA assistant check](https://cla-assistant.io/netdata/netdata?pullRequest=11159)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/netdata/netdata?pullRequest=11159) before we can accept your contribution.
> **mikael** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/netdata/netdata?pullRequest=11159) it.
I've signed the CLA as MikaelUrankar.

**Comment by ilyam8** [comment]
@MikaelUrankar thanks for the contribution! CLA is signed, I clicked on `recheck` and it got fixed.

**Comment by MikaelUrankar** [comment]
```
Some checks were not successful
41 cancelled, 29 queued, 2 neutral, and 5 successful checks 
```
Seems fun but it doesn't explain what failed.

**Comment by ilyam8** [comment]
[Incident with GitHub Actions](https://www.githubstatus.com/incidents/m16jzl31gnqt)

**Comment by diizzyy** [comment]
Seems to be fixed now? I can confirm this working on my aarch64/arm64 box

File: collectors/freebsd.plugin/freebsd_sysctl.c [hunk 1]
```diff
@@ -641,57 +641,63 @@ int do_hw_intcnt(int update_every, usec_t dt) {
...
+                    rrdset_done(st_interrupts);
                 }
-                rrdset_done(st_interrupts);
             }
         }
 
-        old_nintr = nintr;
-    }
+            old_nintr = nintr;
+        }
```
Comment (vlvkobal) [inline collectors/freebsd.plugin/freebsd_sysctl.c:700]:
```suggestion
        old_nintr = nintr;
    }
```

Comment (vlvkobal) [reply collectors/freebsd.plugin/freebsd_sysctl.c:700]: Please fix the indentation. Everything else LGTM.

**Comment by vlvkobal** [comment]
Why have you done the force push? That's more complicated for a reviewer to check what was changed.

**Review by vlvkobal — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**